server/services/conversation.ts

```diff
@@ -114,8 +115,21 @@ export default class ConversationService extends EventEmitter {
         let participantIds: DBObjectId[] = game.galaxy.players.map(p => p._id);
 
         let newConvo = getNewConversation(game, null, name, participantIds);
-
+        
+        let newMessage: ConversationMessage = {
+            _id: mongoose.Types.ObjectId(),
+            fromPlayerId: null,
+            fromPlayerAlias: "Global Chat",
+            message: "Welcome to " + name + "!\n\nThis is the global chat. Any messages sent here will be delivered to all players in the game!\n\nGood Luck, Commanders!",
+            sentDate: moment().utc(),
+            sentTick: game.state.tick,
+            pinned: false,
+            readBy: newConvo.mutedBy!
```

Comment:
As this is a brand new message created at the start of the game, nobody will have been able to mute this.

Set this to an empty array `[]`